src/transformers/configuration_utils.py

```diff
@@ -123,6 +124,12 @@ class PretrainedConfig(object):
         - **num_return_sequences** (:obj:`int`, `optional`, defaults to 1) -- Number of independently computed returned
           sequences for each element in the batch that will be used by default in the :obj:`generate` method of the
           model.
+        - **output_scores** (:obj:`bool`, `optional`, defaults to :obj:`False`) -- Whether the model should return the
+          logits when used for generation
+        - **return_dict_in_generate** (:obj:`bool`, `optional`, defaults to :obj:`False`) -- Whether the model should
+          return a :class:`~transformers.generation_utils.GreedySearchOutput` instead of a :obj:`torch.LongTensor` when
+          generation method is `greedy_search`.
```

Comment:
```suggestion
        - **return_dict_in_generate** (:obj:`bool`, `optional`, defaults to :obj:`False`) -- Whether the model should
          return a :class:`~transformers.file_utils.ModelOutput` instead of a :obj:`torch.LongTensor`.
```
GreedySearchOutput is only one example of generation output, the base type they all share is ModelOutput and the current docstring seem to imply this flag is only used for generate with greedy search, which is not the case.